test/entity_test.rb

```diff
@@ -63,12 +63,6 @@ class Camera
         book.instance_variable_get(:@published).must_equal false
       end
 
-      it 'ignores unknown attributes' do
```

Comment:
@jodosha I am not a fan of graceful unknown handling. Is there any pros of keeping this behaviour?